plaso/parsers/text_plugins/winpca.py

```diff
@@ -0,0 +1,188 @@
+# -*- coding: utf-8 -*-
+"""Parser for Windows Program Compatibility Assistant (PCA) Log files."""
+
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+from plaso.containers import events
+from plaso.lib import errors
+from plaso.parsers import dsv_parser
+from plaso.parsers import manager
+
+
+class WinPCAEventData(events.EventData):
+  """Windows PCA (Program Compatibility Assistant) event data.
+
+  Attributes:
+    body (str): message body.
+    description (str): description of the executable.
+    exit_code (str): final result of the execution.
+    last_execution_time (dfdatetime.DateTimeValues): entry last execution
+        date and time.
+    program_identifier (str): program identifier.
+    run_status (str): execution status.
+    vendor (str): vendor of executed software.
+    version (str): version of executed software.
+  """
+
+  DATA_TYPE = 'windows:pca_log:entry'
+
+  def __init__(self):
+    """Initializes event data."""
+    super(WinPCAEventData, self).__init__(data_type=self.DATA_TYPE)
+    self.body = None
+    self.description = None
+    self.exit_code = None
+    self.last_execution_time = None
+    self.program_identifier = None
+    self.run_status = None
+    self.vendor = None
+    self.version = None
+
+
+# TODO: Check for conflicts with Bodyfile.
```

Comment:
I assume this is a TODO? And assume this needs to be resolved before the code can be merged?

Reply:
Done